include/WRAPPER_SLURM

```diff
@@ -7,6 +7,22 @@
 # make logdir
 if [[ ! -e logs ]]; then mkdir -p logs; fi
 
+# use SNAKEMAKE_PROFILE if one is set in the environment variables
+if [ -z "$LCDBWF_SNAKE_PROFILE" ]; then
```

Comment:
let's change to `LCDBWF_SNAKEMAKE_PROFILE` just to be consistent with the original name

Reply:
changed everywhere